packages/strategies/stress.go

```diff
@@ -39,6 +47,11 @@ func init() {
 		privateKeys[i] = privateKey
 		addresses[i] = crypto.PubkeyToAddress(privateKey.PublicKey)
 	}
+
+	amount := new(big.Int)
+	// 0.1 ether
+	amount.SetString("100000000000000000", 10)
+	transferAmount = common.LeftPadBytes(amount.Bytes(), 32)
```

Comment:
This could be done more efficiently in a single line:
```
transferAmount := common.LeftPadBytes(big.NewInt(100000000000000000).Bytes(), 32)
```
Could also move it out of the `init()` function and just set the value directly when the `transferAmount` variable is defined on line 32